UPDATING.md

```diff
@@ -40,6 +40,13 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Normalize gcp_conn_id for Google Cloud Platform
+
+Previously not all hooks and operators related to Google Cloud Platform use 
+conn_id as parameter for GCP connection. There is currently one parameter 
+which apply to most services. Parameters like ``datastore_conn_id``, ``bigquery_conn_id``,
+``google_cloud_storage_conn_id`` have been deprecated.
```

Comment:
Can you list all affected operators here?

Reply:
I've fixed this list. Previously paths were not complete (missing file name).